### PR TITLE
remove soul docs

### PR DESCRIPTION
**Pull request by tobowers**
* Remove Soul class mentions from the docs
* fix bug at least locally trying to load non-existent file
* upgrade docusaurus

Note this has #96 in it, so once that's merged this will be a smaller PR.

### CODE REVIEW

File: core/src/next/cognitiveFunctions.ts [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
 import { ChatMessageRoleEnum } from "./languageModels";
 import { html } from "common-tags";
 
+
+export const externalDialog = (extraInstructions?: string) => {
+  return instruction(({ entityName: name }: CortexStep) => {
+    return html`
+      ${extraInstructions}
+
+      ${name} should respond as if they were spekaing out loud. The response should be short (as most speech is short), include appropriate verbal ticks, use all caps when SHOUTING, and use punctuation (such as ellipses) to indicate pauses and breaks.
```
Comment (kafischer) [inline core/src/next/cognitiveFunctions.ts:12]:
```suggestion
      ${name} should respond as if they were speaking out loud. The response should be short (as most speech is short), include appropriate verbal ticks, use all caps when SHOUTING, and use punctuation (such as ellipses) to indicate pauses and breaks.
```

**Review by kafischer — CHANGES_REQUESTED** [review]

**Comment by tobowers** [comment]
@kafischer updated this branch to be based on the fixed #96

**Comment by tobowers** [comment]
and now updated to main

**Review by kafischer — APPROVED** [review]

**Comment by kafischer** [comment]
Do you have deploy access here? Or do i need to add you

**Pull request merged**